core/components/atoms/input/Input.tsx

```diff
@@ -223,13 +223,15 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
   );
 
   return (
+    // eslint-disable-next-line
```

Comment:
Instead of disabling the eslint warning please provide the keyboard listener.

Reply:
There is no available keyboard listening event in the Input component design.